system_tests/precompile_test.go

```diff
@@ -627,6 +627,15 @@ func TestArbNativeTokenManager(t *testing.T) {
 	if balanceAfterBurning.Cmp(expectedBalance) != 0 {
 		t.Fatal("expected balance to be", expectedBalance, "got", balanceAfterBurning)
 	}
+
+	// checks sending L2 to L1 value is disabled while native token owners exist
```

Comment:
nitpick: We can add a check that SendTxToL1 succeeds before adding addresses to native token owners.

Reply:
Added.